cypress/helpers/saveToSNOW.js

```diff
@@ -1,6 +1,7 @@
-// import { classificationRequirements,
-//contractType,
-// workflow } from "../helpers/mockedAPICalls/contractDetails";
+// import {
```

Comment:
Since we aren't using this import, we should just remove it

Reply:
Removed the unused code